pymeasure/adapters/visa.py

```diff
@@ -121,18 +121,20 @@ def ask_values(self, command):
         """
         return self.connection.query_values(command)
 
-    def binary_values(self, command, header_bytes=0, dtype=np.float32):
+    def binary_values(self, command, header_bytes=None, dtype=np.float32):
         """ Returns a numpy array from a query for binary data
-
         :param command: SCPI command to be sent to the instrument
         :param header_bytes: Integer number of bytes to ignore in header
```

Comment:
The default case needs to be updated. There are now multiple types being allowed for the same input, which I think will be confusing. I suggest adding a different argument here.

Reply:
A different arg IMHO will be more confising. The docs is surely needs to be updated. BTW, how about dropping python 2 and applying type annotations everywhere?

Reply:
#259